### PR TITLE
Added the curriculum component 

### PR DESCRIPTION
**Pull request by Jitendra797**
- created a stickers folder in the public folder
- created the day component and added that component in the curriculum page

### CODE REVIEW

**Comment by CharanMN7** [comment]
Install the prettier extension, open and save your files again to apply the code formatting.

File: components/day.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "/stickers/web.png",
+    
+ ];
+
+export const Day: React.FC<DayProps> = ({ dayNum, title, content, i }) => {
+  return (
+    <div className="border-b border-r border-l border-dashed p-4 flex flex-nowrap justify-between items-center">
+      {i % 2 === 0 ?  (
+        <>
+          <img
```
Comment (AVidhanR) [inline components/day.tsx:28]: Use the in-built <Image /> component provided by next.js and make sure to add a title to it.

File: components/day.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+              <br />
+              {content.map((line, index) => (
+                <span key={index}>
+                  <span className="pb-2 text-muted-foreground">{line}</span>
+                  <br />
+                </span>
+              ))}
+            </div>
+          </div>
+          <img
```
Comment (AVidhanR) [inline components/day.tsx:64]: Same here use `<Image />` Component.

File: components/day.tsx
[the remark this answers is not in the document]
Comment (Jitendra797) [reply components/day.tsx:1]: Job done 👍

File: components/day.tsx (hunk 1 above)
Comment (Jitendra797) [reply components/day.tsx]: Thanks man 🙌. I changed it

File: components/day.tsx [hunk 3]
```diff
@@ -21,63 +21,35 @@ const stickers = [
...
+      } justify-between items-center`}
+    >
+      <Image
+        src={stickers[i]}
+        alt={`Sticker for day ${dayNum}`}
+        className="w-64 transform -rotate-[10deg]"
+        width={256}
+        height={256}
+      />
+      <div className="flex flex-col items-center gap-4 text-left md:flex-row md:items-start">
```
Comment (CharanMN7) [inline components/day.tsx:38]: I don't think this part is necessary

**Review by CharanMN7 — COMMENTED** [review]
The rotation of images doesn't seem to right, and the order of images and the content for each day card are falling in the same side when the screen size falls under `md` (i.e., `640px`).
Please fix them.

File: components/day.tsx
[the remark this answers is not in the document]
Comment (CharanMN7) [reply components/day.tsx:1]: I see it's fixed for the desktop version of the page. Good work figuring out!

**Pull request merged**